core/trino-spi/src/test/java/io/trino/spi/predicate/TestRange.java

```diff
@@ -42,6 +42,10 @@ public void testInvertedBounds()
         assertThatThrownBy(() -> new Range(Marker.exactly(BIGINT, 1L), Marker.exactly(BIGINT, 0L)))
                 .isInstanceOf(IllegalArgumentException.class)
                 .hasMessage("low must be less than or equal to high");
+
+        assertThatThrownBy(() -> Range.range(BIGINT, 1L, true, 0L, true))
```

Comment:
static import?

Reply:
in the TestRange, i slightly lean towards explicit, since here it _does matter_ which class this method lives in.